### ISSUE TITLE
Break script issues

### ISSUE THREAD

**Issue by GRUFFMAN** [GRUFFMAN/Cat-tastrophe_Game#8]
**Describe the bug**
There are a few issues with the break script.
Upon starting the game there are multiple null exception errors that will spawn throughout gameplay, Not sure how critical they are but it would be good if we fixed up any lose ends that are causing these errors.

On playing the game and testing the colour changing cross hair I noticed that pickupable items (glasses plates) sometimes after being knocked change their layer to 0 or default. I think this is being caused by some sort of bug where the objects are reaching breaking velocity but are not spawning the dead object but the same one they are. this is broken because there objects should still be able to be picked up.

In addition to this it would be good if the broken items spawned on layer 9 (pickups) so that a broken plate can still be picked up, as it is weird that it breaking means you can't pick it up.

(the old fix of making objects break in your mouth still needs to be rectified but that is on a separate thread.

**To Reproduce**
Interact with objects and eventually stuff will go wrong. it doesn't happen every game, but seems to happen enough that I am still seeing it.

Knocking objects around and then they will switch layer to 0 while still being whole.

**Expected behavior**
That everything stays on layer 9.

**Screenshots**

**Desktop (please complete the following information):**

**Additional context**

**Comment by CommanderKeef** [comment]
How dare you assign me work!!?!??!?!?!?

**Comment by CommanderKeef** [comment]
Just kidding I'll get right onto it chief

**Comment by GRUFFMAN** [comment]
Assigning it to you because your break script is above my C# knowledge tbh

**Comment by CommanderKeef** [comment]
I'm very flattered 8^y

**Comment by GRUFFMAN** [comment]
"On playing the game and testing the colour changing cross hair I noticed that pickupable items (glasses plates) sometimes after being knocked change their layer to 0 or default. I think this is being caused by some sort of bug where the objects are reaching breaking velocity but are not spawning the dead object but the same one they are. this is broken because there objects should still be able to be picked up."

Ignore this part of the issue. This was caused by a bad bit of scripting in the knocking script where and object would be set to layer 0 after a timer.

I am not sure why I had this timer in there, my apologies on that part.
